core/add_admin.go

```diff
@@ -26,7 +26,7 @@ func NewAddAdminCmd(ch cmd.CommandHandler) *cmd.Command {
 }
 
 // addAdmin makes an existing user and admin
-func (core *CorePlugin) addAdmin(c cmd.Context) (string, slack.PostMessageParameters) {
+func (core *RocketPlugin) addAdmin(c cmd.Context) (string, slack.PostMessageParameters) {
```

Comment:
Hmmm, that's a good point. I made this change because the VS code Golang plugin was complaining about `core.CorePlugin` "stutter". I can revert this, but would still prefer to rename it, since it leaves a bad impression opening up the codebase and be faced with a bunch of green underlines, especially if you want to encourage plugin development.

`MainPlugin`?